static/js/components/tiles/place_overview_tile.tsx

```diff
@@ -61,7 +61,7 @@ export function PlaceOverviewTile(
       </div>
       {!_.isEmpty(subtopics) && (
         <div className="subtopics-section">
-          <h3>Learn more about {props.place.name}:</h3>
+          <h3>Learn about {props.place.name}:</h3>
```

Comment:
possible to keep the "more" if it's a US place?

Reply:
Now keeps "more" if it's a US place.